data-pipeline.yml

```diff
@@ -0,0 +1,222 @@
+version: 1
+tasks:
+  - ID: commit-retrieval
+    created: {$fromNow: ''}
+    deadline: {$fromNow: '1 hour'}
+    provisionerId: aws-provisioner-v1
+    workerType: tutorial
+    scopes:
+      - docker-worker:cache:services-testing-bugbug-data
+    payload:
+      maxRunTime: 86400
+      image: mozilla/bugbug-commit-retrieval
+
+      artifacts:
+        public/commits.json.xz:
+          path: /data/commits.json.xz
+          type: file
+
+      cache:
+        services-testing-bugbug-data: /cache
+
+      features:
+        taskclusterProxy:
+          true
```

Comment:
We might not need the proxy for this one (as it probably doesn't need to access any secret)